src/Overlay.tsx

```diff
@@ -6,12 +7,30 @@ interface Props {
 }
 
 const Overlay = ({ data }: Props) => {
+
+  
+  /* Define local state hook to store the "user input" data */
+  const [playerData, setPlayerDataSwitch] = React.useState({player1: data.content.players[0],
```

Comment:
import { useState } directly please.

Reply:
this state represents the players data, you should rename `setPlayerDataSwitch` to simply `setPlayerData`